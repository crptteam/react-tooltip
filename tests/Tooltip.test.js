import React from "react";
import { Tooltip } from "../src/";
import { mount } from "enzyme";

describe("Tabs", () => {
  it("Should renders without problems", () => {
    const wrapper = mount(
      <Tooltip
        position={"bottom"}
        content={"I am tooltip"}
      ><div>Hover me</div></Tooltip>
    );

    expect(wrapper).toMatchSnapshot();
  });
});
