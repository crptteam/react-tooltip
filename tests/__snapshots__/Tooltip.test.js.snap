// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tabs Should renders without problems 1`] = `
<WithTheme(Tooltip)
  content="I am tooltip"
  position="bottom"
>
  <Tooltip
    content="I am tooltip"
    position="bottom"
    theme={
      Object {
        "Tooltip": Object {},
      }
    }
  >
    <Wrap
      onMouseEnter={[Function]}
      onMouseMove={[Function]}
      onMouseOut={[Function]}
      theme={
        Object {
          "Tooltip": Object {},
        }
      }
    >
      <div
        className="Wrap-gIyiIM dfZxRk"
        onMouseEnter={[Function]}
        onMouseMove={[Function]}
        onMouseOut={[Function]}
      >
        <TooltipWrap
          left="50%"
          top="calc(100% + 13px)"
          transform="translateX(-50%)"
          visible={false}
        >
          <div
            className="TooltipWrap-jUbtui kdHKHg"
            transform="translateX(-50%)"
          >
            <TopArrow>
              <div
                className="TopArrow-jUEHcQ fIqTSh"
              />
            </TopArrow>
            <TopArrowOverlap>
              <div
                className="TopArrowOverlap-hfoqlJ jSEyTn"
              />
            </TopArrowOverlap>
            I am tooltip
          </div>
        </TooltipWrap>
        <div>
          Hover me
        </div>
      </div>
    </Wrap>
  </Tooltip>
</WithTheme(Tooltip)>
`;
